### ISSUE TITLE
[Feature] Advanced Command Line Navigation

### ISSUE THREAD

**Issue by Math-Dzieu** [sqlitebrowser/sqlitebrowser#2980]
## Describe the new feature
I'm looking for the possibility to open a specific configuration if my sqlitebrowser

Example of what I'm trying to do : 
Open SqliteBrowser at "Browse Data" tab with "Table: FlatChangeView"

I saw that there were keyboard shortcuts that allowed it, but no specific command

#### Example from command prompt
```
"DB Browser for SQLite.exe" "C:\Path\To\The\File"\file.sqlite --tab "Browse Data" --table "FlatChangeView" 
```

## Does this feature exist in another product or project? Please provide a link
I did not find similar features, but for me it seems interesting if you want to create scripts and display the results quickly at the end of script

## Do you have a screenshot? Please add screenshots to help explain your idea.
![image](https://user-images.githubusercontent.com/59688349/153414479-b6e6b7d6-f48e-496c-9d9d-d77ff92968b9.png)

**Comment by chrisjlocke** [comment]
This might be do-able, but if for your own personal use, I would suggest looking into AutoHotKey.
https://www.autohotkey.com/

Its a macro program that could fire up DB4S and 'press the keys' needed to do what you're after.
Not brilliant, but a sort-of solution.

Is this just to view one table?  Or particular figure (eg, the last entry).  Might be easier to knock up a VB application that connects to the database, grabs the record and does what you're after... (unless you need other DB4S specific functions, of course)

**Comment by mgrojo** [comment]
It would be a bug, when this does not work for you:
```
"DB Browser for SQLite" --table my_table my_database.db
```
More options running with `--help` or in the wiki:
https://github.com/sqlitebrowser/sqlitebrowser/wiki/Command-Line-Interface

**Comment by Math-Dzieu** [comment]
> It would be a bug, when this does not work for you:
> 
> ```
> "DB Browser for SQLite" --table my_table my_database.db
> ```
> 
> More options running with `--help` or in the wiki: https://github.com/sqlitebrowser/sqlitebrowser/wiki/Command-Line-Interface

Hello, 
Thank you for your answer but my goal is more to open the application in a specific configuration. In my example command I wanted the application to open on the "Browse Data" tab with the "FlatChangeView" option

![image](https://user-images.githubusercontent.com/59688349/158149855-571bd40f-02b8-417a-ace2-15680be407de.png)

**Comment by Math-Dzieu** [comment]
> This might be do-able, but if for your own personal use, I would suggest looking into AutoHotKey. https://www.autohotkey.com/
> 
> Its a macro program that could fire up DB4S and 'press the keys' needed to do what you're after. Not brilliant, but a sort-of solution.
> 
> Is this just to view one table? Or particular figure (eg, the last entry). Might be easier to knock up a VB application that connects to the database, grabs the record and does what you're after... (unless you need other DB4S specific functions, of course)


Thanks for the tip and sorry for the late reply !

The idea was just to open the application in a specific page to check the correct format of my data, because I open the application after a long process that I let running in the background on my machine. This will simply check and close the application (to avoid interaction and make use as automatic as possible)

**Comment by chrisjlocke** [comment]
[simple table viewer.zip](https://github.com/sqlitebrowser/sqlitebrowser/files/8244291/simple.table.viewer.zip)

OK, this is a 'proof of concept' and hasn't been thoroughly tested or even has any error checking - it was just thrown together.
It should be a Windows application (simple table viewer.exe) and (just tested it!) it will crash if you just run it.  Told you it was proof of concept.

![image](https://user-images.githubusercontent.com/7872218/158162774-77a08e9e-e431-4644-98c2-dcb8a686743f.png)

However, pass it some parameters, and it'll spring into life.

"simple table viewer" /database="put your database in quotes.db" /select="put some fields in quotes or leave blank for all" /table=dont_need_quotes_if_no_spaces /orderBy=dont_need_quotes,for,fields_here

All it has is a refresh button.  
I can improve on this, and plan to as it has (very little) use, but could be useful for someone.

![image](https://user-images.githubusercontent.com/7872218/158163210-8ffb398f-d9af-4610-a6fa-a9ec2d026fdc.png)

So while DB4S can't exactly do what you need, SQLite is small enough that you can use other tools easily to plug into it.

**Comment by mgrojo** [comment]
> Hello, Thank you for your answer but my goal is more to open the application in a specific configuration. In my example command I wanted the application to open on the "Browse Data" tab with the "FlatChangeView" option
> 
> ![image](https://user-images.githubusercontent.com/59688349/158149855-571bd40f-02b8-417a-ace2-15680be407de.png)

Indeed, this command line should do that. Could you try it and report if it didn't work a you expected?

`"DB Browser for SQLite" --table FlatChangeView your_database_file_here.db`

Another option is saving a project file when you are browsing the table and then open the project file from the command line or the file manager. It should restore the state of the application, from when you saved it.
